dart/lib/src/integration.dart

```diff
@@ -6,6 +6,8 @@ import 'sentry_options.dart';
 /// Code that provides middlewares, bindings or hooks into certain frameworks or environments,
 /// along with code that inserts those bindings and activates them.
 abstract class Integration<T extends SentryOptions> {
+  bool called = false;
```

Comment:
if we go for it, I'd prefer adding getter and setter/ to avoid null value, and the getter is read-only.
not sure if adding this here is also something I am really convinced of.